saleor/graphql/shop/types.py

```diff
@@ -270,6 +276,19 @@ def resolve_default_country(_, _info):
             default_country = None
         return default_country
 
+    @staticmethod
+    @permission_required("site.manage_settings")
+    def resolve_default_mail_sender_name(_, info):
+        return info.context.site.settings.default_mail_sender_name
+
+    @staticmethod
+    @permission_required("site.manage_settings")
+    def resolve_default_mail_sender_address(_, info):
+        return (
+            info.context.site.settings.default_mail_sender_address
+            or settings.DEFAULT_FROM_EMAIL
```

Comment:
Is this required? `SiteSettings.default_from_email` property seems to already handle this `or`.